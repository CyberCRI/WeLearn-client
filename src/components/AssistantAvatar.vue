<template>
  <div class="avatar">
    <img alt="chat assistant avatar" src="@/assets/wl_logo.png" />
  </div>
</template>

<style scoped>
.avatar img {
  width: 2rem;
  height: 1.25rem;
}

.avatar {
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0.25rem 0rem;
  border-radius: 50%;
  border: 2px solid var(--primary);
}
@media screen and (min-width: 576px) {
}
</style>
