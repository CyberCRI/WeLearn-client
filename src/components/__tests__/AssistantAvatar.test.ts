import { describe, it, expect } from 'vitest';

import { mount } from '@vue/test-utils';
import AssistantAvatar from '../AssistantAvatar.vue';

describe('AssistantAvatar', () => {
  it('renders properly', () => {
    const wrapper = mount(AssistantAvatar);
    expect(wrapper.element).toMatchSnapshot();
  });
});
