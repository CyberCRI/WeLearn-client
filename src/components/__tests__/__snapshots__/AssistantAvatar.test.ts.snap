// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AssistantAvatar > renders properly 1`] = `
<div
  class="avatar"
  data-v-23b3df74=""
>
  <img
    alt="chat assistant avatar"
    data-v-23b3df74=""
    src="/src/assets/wl_logo.png"
  />
</div>
`;
